modules/ROOT/pages/design-publish.adoc

```diff
@@ -25,10 +32,13 @@ image:apid-publish-dialog-api-specs.png[]
 +
 <1> *Asset version*: The version number to be displayed in the *Version* section of the page that Anypoint Exchange creates for the project.
 <2> *API version*: The version number of the API specification. For a description of this field, see the "Filling in the *API version* Field" section.
-<3> *Business group ID*: The ID of the business unit in Anypoint Platform to which the project belongs.
-<4> *Asset name*: The name that Anypoint Exchange uses to identify the published project.
-<5> *Asset ID*: The ID that Anypoint Exchange uses to identify the published project.
+<3> *Business group ID*: The ID of the business unit in Anypoint Platform to which the project belongs. You should keep the value as is, except in unusual cases.
```

Comment:
"as is" is idiomatic but very common. I can envision my former English conversation students in Japan being stumped by it, but...your call.